lib/pbench/server/database/models/datasets.py

```diff
@@ -211,17 +204,17 @@ def __str__(self) -> str:
 
 
 class MetadataKeyError(MetadataError):
+    """A base class for metadata key errors in the context of Metadata errors
+    that have no associated Dataset.
+
+    It is never raised directly, but may be used in "except" clauses.
     """
-    A base class for metadata key errors in the context of Metadata errors
-    that have no associated Dataset. It is never raised directly, but may
-    be used in "except" clauses.
-    """
+
+    pass
```

Comment:
Given that the docstring constitutes a sufficient body for the class (in the eyes of the Python interpreter), are we sure we want to add the (apparently superfluous) `pass` statement (and its attendant blank line)?

Reply:
Yeah, I wondered about that.  It seems the pass is no longer needed.  Let me try removing them.

Reply:
Fixed.